_posts/2023-10-09-towards_3.0.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: Towards MDAnalysis 3.0
+---
+
+## Overview
+
+MDAnalysis is beginning to plan for our 3.0 release! Following discussions at our first [UGM][], we (the MDAnalysis [Core Developers][]) have decided on a set priorities for this exciting milestone (detailed below). Our primary goals can be divided into several main themes.
+
+
+### A trimmed down core library
+
+As MDAnalysis continues to grow along with the field of molecular simulations, we are aiming to continue to provide a highly functional, easily maintainable package that can be the foundation of a wide variety of downstream tools.
+
+Following the successful introduction of the [MDAKits ecosystem][] (modelled on the SciKit ecosystem, see the [MDAKits paper][]) as part of our [EOSS4][] work, which provides infrastructure for creating [FAIR][] packages that address specific scientific questions in the analysis of molecular simulation, we are aiming to move some more specialized analyses currently present in the MDAnalysis core library to [MDAKits][]. Several analysis modules, including  `hole2`, `encore`, `path-similarity`, `helanal` and `waterdynamics` are being assessed to undergo this transition. Indeed some of these packages are already deployed as [MDAKits][], and are deprecated in the core library.
+
+This change will allow for a reduction in the core dependencies of MDAnalysis and reduce the maintenance burden of specialist analyses in the core library. This will also empower those developing specialist analysis code to rapidly develop and prototype their code in well supported downstream packages. Additionally, core developers will be able to spend more cycles on core competencies, fixing critical issues and planning future improvements, ensuring that MDAnalysis can be easily and effectively maintained for years to come.
+
```

Comment:
Good point. I think, as you say, that moved stuff needs to be maintained, at least for a while. Otherwise MDAKits will be seen mainly as a way to dump code from core and leave it there. I think we have to do the initial maintenance cycle, but totally agree with what you wrote. If someone is willing to take ownership, that great. 

However, what happens if the new owner let it die? That's maybe something to discuss.

Reply:
See discussion on business. Let's wait until Monday if there are other opinions, otherwise we should go ahead with the above and the support of the coredevs.

Reply:
@MDAnalysis/coredevs support the changes above ✅